src/main/java/pages/BasePage.java

```diff
@@ -12,62 +12,62 @@
 import java.time.Duration;
 
 public class BasePage {
-
-    private WebDriverWait wait;
+    
     private static String railway;
 
 
     public void openHomePage() {
         String railwayUrl = Config.getProperty("railway.url");
-        WebDriverConfig.driver.get(railwayUrl);
+        DriverManager.driver.get(railwayUrl);
         // railway = WebDriverConfig.driver.getWindowHandle();
     }
 
     public static void switchToRailway() {
-        WebDriverConfig.driver.switchTo().window(railway);
+        DriverManager.driver.switchTo().window(railway);
     }
 
     public void clickTab(String tabName) {
-        WebDriverWait wait = new WebDriverWait(WebDriverConfig.driver, Duration.ofSeconds(10));
+        WebDriverWait wait = new WebDriverWait(DriverManager.driver, Duration.ofSeconds(10));
         String xpathExpression = String.format("//div[@id='menu']//li/a[span[text()='%s']]", tabName);
         WebElement tabElement = wait.until(ExpectedConditions.elementToBeClickable(By.xpath(xpathExpression)));
         tabElement.click();
     }
 
     public void clickLink(String linkName) {
-        WebDriverWait wait = new WebDriverWait(WebDriverConfig.driver, Duration.ofSeconds(10));
+        WebDriverWait wait = new WebDriverWait(DriverManager.driver, Duration.ofSeconds(10));
         String xpathExpression = String.format("//a[normalize-space()='%s']", linkName);
         WebElement tabElement = wait.until(ExpectedConditions.elementToBeClickable(By.xpath(xpathExpression)));
         tabElement.click();
     }
 
     public static void refreshPage() {
-        WebDriverConfig.driver.navigate().refresh();
+        DriverManager.driver.navigate().refresh();
     }
 
     public static void switchToWindow(String windowHandle) {
-        WebDriverConfig.driver.switchTo().window(windowHandle);
+        DriverManager.driver.switchTo().window(windowHandle);
     }
 
     public static void openNewTab(String url) {
-        WebDriverConfig.driver.switchTo().newWindow(WindowType.TAB);
-        WebDriverConfig.driver.navigate().to(url);
+        DriverManager.driver.switchTo().newWindow(WindowType.TAB);
+        DriverManager.driver.navigate().to(url);
     }
 
+
     public static String getWindowHandle() {
-        return WebDriverConfig.driver.getWindowHandle();
+        return DriverManager.driver.getWindowHandle();
     }
 
     public static void zoomIn(Double zoomNumber) {
-        ((JavascriptExecutor) WebDriverConfig.driver).executeScript(java.lang.String.format("document.body.style.zoom = '%f'", zoomNumber));
+        ((JavascriptExecutor) DriverManager.driver).executeScript(java.lang.String.format("document.body.style.zoom = '%f'", zoomNumber));
     }
 
     public boolean isTabPresent(String tabName) {
-        return WebDriverConfig.driver.findElements(By.linkText(tabName)).size() > 0;
+        return DriverManager.driver.findElements(By.linkText(tabName)).size() > 0;
```

Comment:
it checks the tab displayed in UI, is it? if yes, pls consider on this action. what happened if the element exists in DOM, but it's not visible in UI (hidden by css...)?